sig-node/annual-report-2021.md

```diff
@@ -39,96 +41,101 @@ In future, this will be generated from kubernetes/enhancements kep.yaml files
+- [node-api](https://git.k8s.io/community/sig-node#node-api)
+- [node-feature-discovery](https://git.k8s.io/community/sig-node#node-feature-discovery)
+- [node-problem-detector](https://git.k8s.io/community/sig-node#node-problem-detector)
 -
+[noderesourcetopology-api](https://git.k8s.io/community/sig-node#$noderesourcetopology-api)
+- [security-profiles-operator](https://git.k8s.io/community/sig-node#security-profiles-operator)
 
 ## Working groups
 
-<!--
-In future, this will be generated from delta of sigs.yaml from $YYYY-01-01 to $YYYY-12-31
-Manually visible via `git diff HEAD@{$YYYY-01-01} HEAD@{$YYYY-12-31} -- $sig-id/README.md`
--->
-
-New in $YYYY:
-- [$wg-name](https://git.k8s.io/community/$wg-id/) ([$YYYY report](https://git.k8s.io/community/$wg-id/annual-report-$YYYY.md))
--
+New in 2021:
+- [wg-policy](https://git.k8s.io/community/wg-policy/) ([2021 report](https://git.k8s.io/community/wg-policy/annual-report-2021.md))
+- [wg-structured-logging](https://git.k8s.io/community/wg-structured-logging/) ([2021 report](https://git.k8s.io/community/wg-structured-logging/annual-report-2021.md))
 
-Retired in $YYYY:
-- [$wg-name](https://git.k8s.io/community/$wg-id/) ([$YYYY report](https://git.k8s.io/community/$wg-id/annual-report-$YYYY.md))
--
+Retired in 2021: None
 
 Continuing:
-- [$wg-name](https://git.k8s.io/community/$wg-id/) ([$YYYY report](https://git.k8s.io/community/$wg-id/annual-report-$YYYY.md))
+- [wg-multitenancy](https://git.k8s.io/community/wg-multitenancy/) ([2021 report](https://git.k8s.io/community/wg-multitenancy/annual-report-2021.md))
 -
 
 ## Operational
 
 Operational tasks in [sig-governance.md]:
 
-- [ ] [README.md] reviewed for accuracy and updated if needed
-- [ ] [CONTRIBUTING.md] reviewed for accuracy and updated if needed
+- [x] [README.md] reviewed for accuracy and updated if needed
+- [x] [CONTRIBUTING.md] reviewed for accuracy and updated if needed
       (or created if missing and your contributor steps and experience are different or more
       in-depth than the documentation listed in the general [contributor guide] and [devel] folder.)
-- [ ] Subprojects list and linked OWNERS files in [sigs.yaml] reviewed for accuracy and updated if needed
-- [ ] SIG leaders (chairs, tech leads, and subproject owners) in [sigs.yaml] are accurate and active, and updated if needed
-- [ ] Meeting notes and recordings for 2021 are linked from [README.md] and updated/uploaded if needed
+- [x] Subprojects list and linked OWNERS files in [sigs.yaml] reviewed for accuracy and updated if needed
+- [x] SIG leaders (chairs, tech leads, and subproject owners) in [sigs.yaml] are accurate and active, and updated if needed
+- [x] Meeting notes and recordings for 2021 are linked from [README.md] and updated/uploaded if needed
 - [ ] Did you have community-wide updates in 2021 (e.g. community meetings, kubecon, or kubernetes-dev@ emails)? Links to email, slides, or recordings:
```

Comment:
Not sure on this one - If anyone has any items add them as a comment here and I'll update